packages/core/template_project/app/build.gradle

```diff
@@ -197,4 +197,4 @@ dependencies {
         implementation '<%= dep %>'
     <% } %>
     implementation 'com.google.androidbrowserhelper:androidbrowserhelper:2.0.0'    
```

Comment:
When adding the dependency via FeatureManager, this line will duplicate the dependency.